demo/client/backend/auction.json

```diff
@@ -0,0 +1,203 @@
+		"id": 1,
+		"name": "Elbograd",
+		"isHighDemand": false,
+		"minPrice": 10000,
+		"channels": [{
+			"id": 1,
+			"name": "a",
+			"impairment": 13
+		}, {
+			"id": 2,
+			"name": "b",
+			"impairment": 0
+		}, {
+			"id": 3,
+			"name": "c",
+			"impairment": 5
+		}, {
+			"id": 4,
+			"name": "d",
+			"impairment": 6
+		}, {
+			"id": 5,
+			"name": "e",
+			"impairment": 0
+		}, {
+			"id": 6,
+			"name": "f",
+			"impairment": 12
+		}, {
+			"id": 7,
+			"name": "g",
+			"impairment": 20
+		}, {
+			"id": 8,
+			"name": "h",
+			"impairment": 5
+		}, {
+			"id": 9,
+			"name": "i",
+			"impairment": 0
+		}, {
+			"id": 10,
+			"name": "j",
+			"impairment": 10
+		}]
+	}, {
+		"id": 2,
+		"name": "Mudberg",
+		"isHighDemand": false,
+		"minPrice": 6000,
+		"channels": [{
+			"id": 1,
+			"name": "a",
+			"impairment": 20
+		}, {
+			"id": 2,
+			"name": "b",
+			"impairment": 25
+		}, {
+			"id": 3,
+			"name": "c",
+			"impairment": 10
+		}, {
+			"id": 4,
+			"name": "d",
+			"impairment": 5
+		}, {
+			"id": 5,
+			"name": "e",
+			"impairment": 0
+		}, {
+			"id": 6,
+			"name": "f",
+			"impairment": 0
+		}, {
+			"id": 7,
+			"name": "g",
+			"impairment": 0
+		}, {
+			"id": 8,
+			"name": "h",
+			"impairment": 0
+		}, {
+			"id": 9,
+			"name": "i",
+			"impairment": 15
+		}, {
+			"id": 10,
+			"name": "j",
+			"impairment": 23
+		}]
+	}, {
+		"id": 3,
+		"name": "Deserton",
+		"isHighDemand": false,
+		"minPrice": 6000,
+		"channels": [{
+			"id": 1,
+			"name": "a",
+			"impairment": 0
+		}, {
+			"id": 2,
+			"name": "b",
+			"impairment": 0
+		}, {
+			"id": 3,
+			"name": "c",
+			"impairment": 6
+		}, {
+			"id": 4,
+			"name": "d",
+			"impairment": 10
+		}, {
+			"id": 5,
+			"name": "e",
+			"impairment": 5
+		}, {
+			"id": 6,
+			"name": "f",
+			"impairment": 0
+		}, {
+			"id": 7,
+			"name": "g",
+			"impairment": 0
+		}, {
+			"id": 8,
+			"name": "h",
+			"impairment": 0
+		}, {
+			"id": 9,
+			"name": "i",
+			"impairment": 0
+		}]
+	}, {
+		"id": 4,
+		"name": "Phlimsk",
+		"isHighDemand": false,
+		"minPrice": 4000,
+		"channels": [{
+			"id": 1,
+			"name": "a",
+			"impairment": 0
+		}, {
+			"id": 2,
+			"name": "b",
+			"impairment": 0
+		}, {
+			"id": 3,
+			"name": "c",
+			"impairment": 0
+		}, {
+			"id": 4,
+			"name": "d",
+			"impairment": 8
+		}, {
+			"id": 5,
+			"name": "e",
+			"impairment": 25
+		}, {
+			"id": 6,
+			"name": "f",
+			"impairment": 30
+		}]
+	}],
+	"bidders": [{
+		"id": 1,
+		"displayName": "A-Telecom",
+		"principle": {
+			"mspid": "org1",
+			"dn": "bidder@org1"
+		}
+	}, {
+		"id": 2,
+		"displayName": "B-Net",
+		"principle": {
+			"mspid": "org2",
+			"dn": "bidder@org2"
+		}
+	}, {
+		"id": 3,
+		"displayName": "C-Mobil",
+		"principle": {
+			"mspid": "org3",
+			"dn": "bidder@org3"
+		}
+	}],
+	"initialEligibilities": [{
+		"bidderId": 1,
+		"number": 10000
+	}, {
+		"bidderId": 2,
+		"number": 10000
+	}, {
+		"bidderId": 3,
+		"number": 10000
+	}],
+	"owner": {}
```

Comment:
> I don't think you need any owner as this is the template for createAuction (which doesn't have an owner, only getAuctionDetails does ...)

Will check with Marcus and fix accordingly since this is json content was  defined and used by UI component.

Reply:
@sjanakir2015  "owner" can be removed here